common/lib/xmodule/xmodule/js/src/video/09_completion.js

```diff
@@ -0,0 +1,142 @@
+(function(define) {
+    'use strict';
+    define('video/09_completion.js', [], function() {
+    /**
+     * Completion handler
+     * @exports video/09_completion.js
```

Comment:
By looking at the other files in video.  They seem to be roughly ordered by dependency.  completion is required by 10_main, so it needed to be less than 10, but I figured not much else depends on it, so it can be added late in the process.

I didn't really understand why the numbering is needed here, but I figured it was better to be consistent with the subsystem I'm working in.